client/src/paths/Trash.tsx

```diff
@@ -41,57 +31,46 @@ export function Trash() {
   }, []);
 
   const fetcher = useFetcher();
-  const navigate = useNavigate();
 
-  const headingText = "My Trash";
+  const titleIcon = (
+    <Tooltip label={"My Trash"}>
+      <Box>
+        <Icon
+          as={LuTrash2}
+          boxSizing="content-box"
+          width="24px"
+          height="24px"
+          mr="0.5rem"
+          verticalAlign="middle"
+          aria-label={"My Trash"}
+        />
+      </Box>
+    </Tooltip>
+  );
+
+  const headingText = (
+    <EditableName
```

Comment:
Unless you want people to change the name of the trash (which doesn't seem to work), you can get rid of  `<EditableName>`.

It seems like `<EditableName>` might be used in other places where we don't actually want to edit the name. Fix it here or add an issue?